.gitignore

```diff
@@ -3,6 +3,8 @@ __pycache__/
 *.py[cod]
 *$py.class
 
+tools/release/issue.md
```

Comment:
Missing file ?

Reply:
I do not understand. I put output of `generate_issue_content.sh` in this file.

Reply:
So you are doing `sh generate_issue_content.sh > tools/release/issue.md` ? But where ? It's not documented anywhere, or maybe I'm missing something ?

Reply:
it is not documented, as I do not add yet any readme with documentation.

Reply:
ok.